registration/api/v2/serializers.py

```diff
@@ -338,9 +339,10 @@ def validate_external_domain(self, value):
         """
         Prevent users from registering with an external domain which was or currently in use.
         """
-        is_changed = self.instance and self.instance.external_domain == self.initial_data.get("external_domain")
+        is_new_instance = self.instance is None
+        is_changed = not is_new_instance and self.instance.external_domain == self.initial_data.get("external_domain")
```

Comment:
Is checking equality correct here? `self.instance.external_domain == self.initial_data.get("external_domain")` Shouldn't it be checking if it's not equal (`is_changed`)?

Reply:
Well, yes. That should be `!=` instead of `==`. Fixing, adding test and the I'll do the merge 😊